drivers/SmartThings/matter-button/src/test/test_matter_button_parent_child.lua

```diff
@@ -242,48 +202,6 @@ test.register_message_test(
   }
 )
 
-test.register_message_test(
```

Comment:
Shouldn't we keep these test cases (and others in this file) but just change the device that is being used in the test case to be one child devices that still exists? I understand we are on a tight schedule so we can remove these here and just add them back again with the adjusted mock devices after the deploy if needed.

Reply:
OK I believe I addressed the comment and restored the testing of the unique functionality that previously existed on buttons 4/5